scripts/xss_linter.py

```diff
@@ -2003,7 +2003,7 @@ def _is_django_template(self, mako_template):
             True if this is really a Django template, and False otherwise.
 
         """
-        if re.search('({%.*%})|({{.*}})', mako_template) is not None:
+        if re.search('({%.*%})|({{.*}})|({#.*#})', mako_template) is not None:
```

Comment:
Thanks.  However, I see the second one addressed, but not the first.  Am I just missing something, or do you not like that suggestion?

Also, would you mind running the script before/after on edx-platform and ensure that we either have the same number of mako results, or that you are removing some that shouldn't have been there in the first place?  Note that if you are changing the results, you may want to drop the thresholds as well.